dotCMS/src/integration-test/java/com/dotmarketing/portlets/categories/business/CategoryFactoryTest.java

```diff
@@ -0,0 +1,273 @@
+package com.dotmarketing.portlets.categories.business;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+import com.dotcms.IntegrationTestBase;
+import com.dotmarketing.business.FactoryLocator;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.dotmarketing.portlets.categories.model.Category;
+import java.util.List;
+import org.jetbrains.annotations.NotNull;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+/***
+ * Category Factory Test
+ */
+public class CategoryFactoryTest extends IntegrationTestBase {
+
+    private static CategoryFactory categoryFactory;
+
+    @BeforeClass
+    public static void prepare() throws Exception {
+        categoryFactory = FactoryLocator.getCategoryFactory();
+    }
+
+    /**
+     * Method to test: {@link CategoryFactory#save(Category)}
+     * Given Scenario: attempt to save an empty category object
+     * ExpectedResult: Should throw a DotDataException
+     * @throws DotSecurityException
+     * @throws DotDataException
+     */
+    @Test(expected = DotDataException.class)
+    public void Test_Save_Empty_Category_Expect_Exception() throws DotDataException {
+        Category category = new Category();
+        categoryFactory.save(category);
+    }
+
+    /**
+     * Method to test: {@link CategoryFactory#save(Category)}
+     * Given Scenario: attempt to save an empty category object
+     * ExpectedResult: Should throw a DotDataException
+     * @throws DotSecurityException
+     * @throws DotDataException
+     */
+    @Test
```

Comment:
If an exception is expected here, why is not declared as a parameter on the `@Test` annotation?

Reply:
Doc is wrong. Is a Copy of the one above this test

Reply:
done